packages/base/src/3dview/mainview.tsx

```diff
@@ -372,11 +397,15 @@ export class MainView extends React.Component<IProps, IStates> {
         this._clippingPlaneMesh.position.z - this._clippingPlane.normal.z
       );
     }
+    if (this._viewHelper.animating) {
+      this._viewHelper.update(this._delta);
+    }
 
     this._controls.update();
     this._renderer.setRenderTarget(null);
-    this._renderer.clearDepth();
+    this._renderer.autoClear = false;
```

Comment:
What I meant in https://github.com/jupytercad/JupyterCAD/pull/440#discussion_r1782971785 

was that this line should be:

```suggestion
    this._renderer.clear();
```

And that `autoClear` should be set to false only upon creation of the renderer (no need to set it on every frame). So maybe adding `this._renderer.autoClear = false;` around line 228 from this file

Reply:
Thanks @martinRenou, done in [`5d8dc84`](https://github.com/jupytercad/JupyterCAD/pull/440/commits/5d8dc84cfc7ce7e02be3783b75744403efabecfb)